testing/test_aux.out

```diff
@@ -68,9 +70,9 @@ RESULT info communication.type.broadcast Broadcast packets received. Unicast pac
 RESULT skip connection.network.ntp_support No NTP packets received.
 RESULT skip connection.network.ntp_update Not enough NTP packets received.
 RESULT pass connection.mac_oui Manufacturer: Google found for address 3c:5a:b4:1e:8f:0a
-dhcp requests 1 1 1 1
+dhcp requests 1 1 1 0
```

Comment:
why is this happening?  Is this an intentional change?  What's the 1->0 request that's going away?

Reply:
@grafnu This is the output when ipaddr is enabled. In the current master branch ipaddr is not enabled.